since 3.8, egrep and fgrep print a warning which can cause many issues with
programs not expecting this behaviour

--- a/src/egrep.sh
+++ b/src/egrep.sh
@@ -1,4 +1,3 @@
 #!@SHELL@
 cmd=${0##*/}
-echo "$cmd: warning: $cmd is obsolescent; using @grep@ @option@" >&2
 exec @grep@ @option@ "$@"
